rxbindings/src/main/java/com/amplifyframework/rx/RxPredictionsCategoryBehavior.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.rx;
+
+import android.graphics.Bitmap;
+import androidx.annotation.NonNull;
+
+import com.amplifyframework.predictions.PredictionsCategoryBehavior;
+import com.amplifyframework.predictions.PredictionsException;
+import com.amplifyframework.predictions.models.IdentifyAction;
+import com.amplifyframework.predictions.models.LanguageType;
+import com.amplifyframework.predictions.options.IdentifyOptions;
+import com.amplifyframework.predictions.options.InterpretOptions;
+import com.amplifyframework.predictions.options.TextToSpeechOptions;
+import com.amplifyframework.predictions.options.TranslateTextOptions;
+import com.amplifyframework.predictions.result.IdentifyResult;
+import com.amplifyframework.predictions.result.InterpretResult;
+import com.amplifyframework.predictions.result.TextToSpeechResult;
+import com.amplifyframework.predictions.result.TranslateTextResult;
+
+import io.reactivex.Single;
+
+/**
+ * An Rx-idiomatic expression of the {@link PredictionsCategoryBehavior}s.
+ */
+public interface RxPredictionsCategoryBehavior {
+    /**
+     * Converts the supplied text to speech.
```

Comment:
EXTREMELY MINOR nit:
convert"s"/translate"s" vs identify/interpret

Reply:
> EXTREMELY MINOR nit:
> 
> convert"s"/translate"s" vs identify/interpret

Thanks, good catch. I’ll update.

Reply:
Update: we talked offline and weren't positive which verb tense was correct. To document our finding here, [Oracle](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html) says:
> Use 3rd person (descriptive) not 2nd person (prescriptive). The description is in 3rd person declarative rather than 2nd person imperative.